### PR TITLE
feat: Revamped Apps area with Batch Exports

### PR DESCRIPTION
**Pull request by benjackwhite**
## Problem

"Apps" is changing to have different approaches depending on the type of thing happening, in particular Batch Exports. In addition, the existing Apps page relies heavily on AntD.

## Changes

* Reworks the Apps page to have a single tab for all traditional Apps
* Adds a new tab for batch exports
* Logs are moved to the "app" view which also now has a configure button

|Before|After|
|----|----|
| Screenshot 2023-08-18 at 15 17 04 | Screenshot 2023-08-18 at 15 15 27 |
| Screenshot 2023-08-18 at 15 14 49 | Screenshot 2023-08-18 at 15 15 09 |
| Screenshot 2023-08-18 at 15 14 43  |  |
| Screenshot 2023-08-18 at 15 15 01 | Screenshot 2023-08-18 at 15 15 16 |
|  | Screenshot 2023-08-18 at 15 20 18 |

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

## TODO
- [x] Re-ordering of enabled apps
- [x] Add missing error states to app view
- [x] "Advanced" options for installing your own apps
- [x] Better metrics page to configure apps

### CODE REVIEW

**Review by tomasfarias — APPROVED** [review]
LGTM although I can't comment much on the frontend bits (which is actually most of the bits). From a subjective aesthetic perspective I love it!

Looking forward to toggling this on for our team and getting some early feedback.

File: frontend/src/scenes/plugins/tabs/apps/AppsTab.tsx [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    } = useValues(pluginsLogic)
+
+    const officialPlugins = useMemo(
+        () => filteredUninstalledPlugins.filter((plugin) => plugin.maintainer === 'official'),
+        [filteredUninstalledPlugins]
+    )
+    const communityPlugins = useMemo(
+        () => filteredUninstalledPlugins.filter((plugin) => plugin.maintainer === 'community'),
+        [filteredUninstalledPlugins]
+    )
```
Comment (mariusandra) [inline frontend/src/scenes/plugins/tabs/apps/AppsTab.tsx:43]: These could go inside `pluginsLogic` as selectors.

**Review by mariusandra — APPROVED** [review]
Could nit on some things (left one comment inline), but I generally think this is good to merge as is 👍 

However... adding an app blanked the screen for me:



Luckily the fix was quite easy --> we were taking the user to the old tab. 

This "empty state" screen also looks odd with the horizontal line in the middle, but it's so much better than what we have now, that I'll pretend not to see it.

image

**Comment by tomasfarias** [comment]
I've pushed a commit to attempt to solve the conflicts with master.

**Comment by tomasfarias** [comment]
I've rebased on master and pushed a commit to address the conflicts. Hopefully I didn't mess things up. If I did, I've saved a copy of the branch before the rebase.

**Comment by tomasfarias** [comment]
Locally testing this looks good after my rebase, so I'll be merging!

**Pull request merged**